Katas UI review - Multiqubit systems (#1813)
microsoft/qsharp: katas/content/multi_qubit_systems/bell_state_change_1/solution.md

You recognize that the goal is another Bell state. In fact, it is one of the four Bell states.

> You remember from the Single-Qubit Gates kata that the Pauli Z gate will change the state of the $\ket{1}$ basis state of a single qubit, so this gate seems like a good candidate for what you want to achieve. This gate leaves the sign of the $\ket{0}$ basis state of a superposition unchanged, but flips the sign of the $\ket{1}$ basis state of the superposition.

Reviewer: This is a bold statement when switched to "you"... I'd modify
```suggestion
You have seen in the Single-Qubit Gates kata that the Pauli Z gate will change the state of the $\ket{1}$ basis state of a single qubit, so this gate seems like a good candidate for what you want to achieve. This gate leaves the sign of the $\ket{0}$ basis state of a superposition unchanged, but flips the sign of the $\ket{1}$ basis state of the superposition.
```

> Don't forget that the Z gate acts on only a single qubit, and you have two here.

Reviewer 2: I'm noticing that in this file, as well as the solution.md files under bell_state_change_2 and bell_state_change_3, don't use the math context when referencing the Z and X gates. Is that okay here? I guess if it is referenced like "Pauli Z gate", I'd leave it as plain text, but when it is just "Z gate", maybe it should be "$Z$ gate" instead for consistency. This is a small nit though, so if you have rational as to why it should be plain text, I'm happy to close this comment.

Reviewer: My intended style here is pretty much as you suggest, "Pauli X gate" feels better in plain text but "$X$ gate" in math. A lot of these katas were migrated in different times, so there are definitely places where I missed inconsistencies!

@SoniaLopezBravo Can you please take a look at this comment and update the files modified in this PR accordingly? Thank you!

Author: @tcNickolas should this apply to identifying letters in systems, such as "system A and system B". Should they be "system $A$ and system $B$"? Thanks!!

Reviewer: I don't have a preference on this one
